backend/src/LibTreeSitter/LibTreeSitter.fsproj

```diff
@@ -55,6 +55,14 @@
     <EmbeddedResource Include="lib/tree-sitter-macos-arm64.dylib" LogicalName="tree-sitter.dylib" />
     <EmbeddedResource Include="lib/tree-sitter-darklang-macos-arm64.dylib" LogicalName="tree-sitter-darklang.so" />
```

Comment:
was it intentional to use `.so` instead of `.dylib` for the logical name of the embedded resource on macOS

Reply:
no